translate/storage/test_properties.py

```diff
@@ -95,6 +96,90 @@ def test_rich_set(self):
         pass
 
 
+class TestGwtProp(test_monolingual.TestMonolingualStore):
+    StoreClass = properties.propfile
+
+    def propparse(self, propsource, personality="gwt", encoding=None, sourcelanguage=None, targetlanguage=None):
+        """helper that parses properties source without requiring files"""
+        dummyfile = wStringIO.StringIO(propsource)
+        propfile = properties.propfile(None, personality, encoding)
+        if sourcelanguage:
+            propfile.sourcelanguage = sourcelanguage
+        if targetlanguage:
+            propfile.targetlanguage = targetlanguage
+        propsrc = dummyfile.read()
+        dummyfile.close()
+        propfile.parse(propsrc)
+        propfile.makeindex()
+        return propfile
+
+    def propregen(self, propsource):
+        """helper that converts properties source to propfile object and back"""
+        return self.propparse(propsource).__bytes__()
+
+    def test_simpledefinition(self):
+        """checks that a simple properties definition is parsed correctly"""
+        propsource = 'test_me=I can code!'
+        propfile = self.propparse(propsource)
+        assert len(propfile.units) == 1
+        propunit = propfile.units[0]
+        assert propunit.name == "test_me"
+        assert propunit.source == "I can code!"
+
+    def test_doubledefinition(self):
+        """checks that a double properties definition is parsed correctly"""
+        propsource = 'test_me=I can code!\ntest_me[one]=I can code single!'
+        propfile = self.propparse(propsource)
+        assert len(propfile.units) == 1
+        propunit = propfile.units[0]
+        assert propunit.name == "test_me"
+        assert propunit.source.strings == ["I can code!", "I can code single!"]
```

Comment:
This test fails on Python 3.4: https://travis-ci.org/translate/translate/jobs/514318431#L946